File: Selenium/Activities/Activity9.java
package activities;

import java.time.Duration;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.WebElement;
import org.openqa.selenium.chrome.ChromeDriver;
import org.openqa.selenium.support.ui.ExpectedConditions;
import org.openqa.selenium.support.ui.WebDriverWait;

public class Activity9 {
	public static void main(String[] args) {
		
		WebDriver driver = new ChromeDriver();
		WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(10));
		 
		driver.get("https://v1.training-support.net/selenium/ajax");
		System.out.println("Home page title: " + driver.getTitle());
		
		driver.findElement(By.xpath("//button[text()='Change Content']")).click();

        wait.until(ExpectedConditions.visibilityOfElementLocated(By.tagName("h1")));
        String messageCheck = driver.findElement(By.tagName("h1")).getText();
        System.out.println(messageCheck);
        
        WebElement mesage1= driver.findElement(By.tagName("h3"));      
        System.out.println(mesage1.getText());
        
        wait.until(ExpectedConditions.textToBePresentInElementLocated(By.tagName("h3"), "I'm late!"));
        WebElement mesage2= driver.findElement(By.tagName("h3"));      
        System.out.println(mesage2.getText());
	
	}

}

File: Selenium/Activities/activity9.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.support.ui import WebDriverWait
from selenium.webdriver.support import expected_conditions
with webdriver.Chrome() as driver:

    driver.get("https://v1.training-support.net/selenium/ajax")
    print("Home page title: ", driver.title)

    wait = WebDriverWait(driver, 10)

    driver.find_element(By.xpath("//button[text()='Change Content']")).click()
 
    wait.until(expected_conditions.text_to_be_present_in_element((By.TAG_NAME, "h1"), "HELLO!"))
    message1 = driver.find_element(By.TAG_NAME, "h1").text
    print(message1)

    wait.until(expected_conditions.text_to_be_present_in_element((By.TAG_NAME, "h3"), "I'm late!"))
    message2 = driver.find_element(By.TAG_NAME, "h3").text

    print(message2)